cli/testdata/coder_--help.golden

```diff
@@ -0,0 +1,67 @@
+Coder v0.0.0-devel — A tool for provisioning self-hosted development environments with Terraform.
+
+Usage:
+  coder [flags]
+
+  coder [command]
+
+Get Started:
+  - Start a Coder server:                                                       
+
+      [;m$ coder server[0m 
```

Comment:
nit: you may consider disabling colors for readability purposes. I assume that the major goal of this change request is to check the CLI text output.

Reply:
They should already be since stdout is not a tty, so seems like a separate issue 😅. (You're absolutely right that it _shouldn't_ be output since we only care about the text.)